### PR TITLE
Add support for Less

### PR DESCRIPTION
**Pull request by vanillajonathan**

### CODE REVIEW

**Comment by bwinton** [comment]
So, I've made some changes to the build file that I forgot to push, to make the code more similar, and I'm afraid those broke your changes here. Would you mind re-implementing them on top of the new structure?

**Comment by vanillajonathan** [comment]
@bwinton Alright. I think I resolved the merge conflict. :)

File: colors/build.js [hunk 1]
```diff
@@ -25,6 +25,13 @@ const formats = {
     'footer': '}\n',
     'ext': 'css'
   },
+  'less': {
+    'output': [`/* Firefox Colors Less Variables v${metadata.version} */
+
+`],
+    'formatter': (color, variant, value) => `@${color}-${variant}: ${value};\n`,
+    'ext': 'less'
+  },
```
Comment (bwinton) [inline colors/build.js:34]: This seems like an easier change to review. 😉

The one thing I would ask for is you to generate the less file and add it in to this commit, but other than that, I'm happy!

**Review by bwinton — APPROVED** [review]

File: colors/build.js (hunk 1 above)
Comment (vanillajonathan) [reply colors/build.js:34]: Indeed.

I don't have Node.js installed. Can you generate the .less file?

Comment (bwinton) [reply colors/build.js:34]: Okay, I'll merge this, and generate those in a new commit.  Thanks!

**Comment by bwinton** [comment]
And it's generated and pushed to npm.

**Pull request merged**